apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -565,15 +568,18 @@ static int ws_test_base(void *arg)
 			}
 		}
 		int loopcount = 0;
+		int buf_msg_len = 0;
+
+		len = 0;
+		memset(param->response->message, 0, WS_TEST_CONF_MAX_MESSAGE_SIZE);
 		while (!read_finish) {
-			PRNT("Receive start");
-			memset(param->response->message, 0, WS_TEST_CONF_MAX_MESSAGE_SIZE);
+			PRNT("Receive start \n");
 			if (param->tls) {
 				len = mbedtls_ssl_read(&(client_tls->tls_ssl),
 									   (unsigned char *)param->response->message,
 									   WS_TEST_CONF_MAX_MESSAGE_SIZE);
 			} else {
-				len = recv(sockfd, param->response->message,
+				len = recv(sockfd, param->response->message + buf_msg_len,
```

Comment:
If total message size is larger than WS_TEST_CONF_MAX_MESSAGE_SIZE, a overflow maybe happen. doesn't it?

Reply:
resolved